code_generation/enum.h

```diff
@@ -84,6 +84,13 @@ class KODE_EXPORT Enum
      */
     void setIsQENUM(bool qenum = true);
 
+    /**
+     * @brief setIsQENUM
+     * This method can be used to generate typedef for the given enum
```

Comment:
Maybe document that this only makes sense for C code?
It's very unusual in C++ code (and probably deprecated?)